compute/k8s-services/main.tf

```diff
@@ -523,7 +530,7 @@ module "crossplane" {
   crossplane_view_service_accounts  = var.crossplane_view_service_accounts
   crossplane_metrics_enabled        = var.crossplane_metrics_enabled
   crossplane_aws_iam_role_name      = var.crossplane_aws_iam_role_name
-  eks_openid_connect_provider_url   = var.eks_openid_connect_provider_url
```

Comment:
You should remove the var from k8s-services vars.tf with no longer referring to it as it is redundant

Reply:
Strangely the code shown above suggest i've entirely removed that section.

The line itself is still valid but instead of referencing a var it simply pulls it directly from the datasourcenow.

Reply:
you have replaced var.eks_openid_connect_provider_url with data.aws_eks_cluster.eks.identity[0].oidc[0].issuer
but you have not deleted the eks_openidconnect_provider_url variable from the vars.tf file as far as i can see at https://github.com/dfds/infrastructure-modules/blob/551aff5f9fc0f29d9b52ce353e754dc4669d5464/_sub/compute/helm-crossplane/vars.tf#L70 which is no longer required since you removed the use of it so it is sat there unused/redundant